### ISSUE TITLE
Prettier breaks Concordion HTML Templates which then fails CI Tests

### ISSUE THREAD

**Issue by adarro** [truthencode/ddo-calc#88]
Example failing build [Circle CI build](https://app.circleci.com/pipelines/github/truthencode/ddo-calc/356/workflows/e2cdcae4-92f0-4942-b5cf-87b290adfa9e/jobs/332)
Formatting via Trunk delegates HTML linting /  Formatting to Prettier.
Prettier then automatically lowercases the DOCTYPE declaration in HTML templates used by Concordion.
Concordion then fails with an initialization error when executing the template. (error similar to https://github.com/concordion/concordion/issues/15)  #https://github.com/prettier/prettier/pull/7391 and https://github.com/prettier/prettier/issues/6502 as it expects an Uppercase DOCTYPE.

Fixes:
- Manually revert DOCTYPE after formatting (fragile /hacky)
- Add / maintain custom exclusion configurations for affected templates. (fragile / hacky)
- Add blanket exclusion for all Concordion HTML template folders (incurring loss of 99% point of using Prettier)
- Convert existing HTML templates to their Markdown equivalent. (Some advanced features are only available using @FullOGNL and HTML.)
- Create Issue @ Concordion to make their tool case-insensitive. 
- Add / create parser to undo Prettier's opinion.

The official stance is that DOCTYPE in this context _should_ be case-insensitive.  However, each tool explicitly utilizes a different option.

Current work-around:
Trunk check / fmt should only affect new and modified code, so once it's manually committed, it _shouldn't_ break.  
Forgetting to manually implement the work-around simply results in wasted cycles and an assumption the less-than-obvious stack-trace will trigger recollection of the needed fix.

**Comment by eligrey** [comment]
> The official stance is that DOCTYPE in this context should be case-insensitive

This is not correct. You are using an XML context, and DOCTYPE is case sensitive in that context.
